esmvaltool/utils/testing/regression/compare.py

```diff
@@ -30,20 +31,39 @@
 """Files to ignore when comparing results."""
 
 IGNORE_GLOBAL_ATTRIBUTES: tuple[str, ...] = (
+    'ancestors',
+    'auxiliary_data_dir',
     'creation_date',
     'history',
     'provenance',
     'software',
+    'version',
 )
 """Global NetCDF attributes to ignore when comparing."""
 
+IGNORE_VARIABLE_ATTRIBUTES: tuple[str, ...] = IGNORE_GLOBAL_ATTRIBUTES
+"""Variable NetCDF attributes to ignore when comparing."""
+
+IGNORE_VARIABLES: tuple[str, ...] = (
+    'temp_list',  # used by perfmetrics diagnostics to save absolute paths
```

Comment:
What kind of absolute paths are these?

Reply:
Absolute paths to output files created by the tool:

```bash
> ncdump ./recipe_perfmetrics_CMIP5_20220701_072823/work/toz/grading_global/RMSD.nc                                                                                                                                                                             3540ms
netcdf RMSD {
dimensions:
        ncl0 = 1 ;
        ncl1 = 120 ;
variables:
        char temp_list(ncl0, ncl1) ;
                temp_list:_FillValue = "" ;
data:

 temp_list =
  "/scratch/b/b381943/esmvaltool_output/recipe_perfmetrics_CMIP5_20220701_072823/work/toz/grading_global/RMSD_toz-global.nc" ;
}
```

Reply:
OK, I think this is also some problem with the interface between diagnostic scripts that have another diagnostic as an ancestor, right?

Reply:
Yes, exactly. I think this is a remnant from version 1 of ESMValTool.

Reply:
Created an issue #2714